extension/src/popup/constants/routes.ts

```diff
@@ -12,5 +12,5 @@ export enum ROUTES {
   recoverAccountSuccess = "/recover-account-success",
   unlockBackupPhrase = "/unlock-backup-phrase",
   displayBackupPhrase = "/display-backup-phrase",
-  viewPublicKey = "/viewPublicKey",
+  viewPublicKey = "/view-public-key",
```

Comment:
just making this route match the same casing as the others.

Also, fun fact, I just learned that this type of casing is often called kebab-casing 🍖